docs/DesigningProcesses.md

```diff
@@ -126,7 +126,16 @@ Variable once defined cannot be overwritten.
 
 ![split graph](img/variable_window.png)
   
-  
+## Map Variable 
+
+Evaluates `Expression` and assigns to `MapVariable.FieldName`. Allows
+to assign many of `Expression`.
+
+![split graph](img/variable_object_window.png)
+
+Doesn't change records flow. 
+Have to have exactly one output.
+MapVariable once defined cannot be overwritten.  
```

Comment:
What do you mean by this? Do you mean that defined variable cannot be overwritten by subsequent operations? What will happen if smb try to do this?

Reply:
When I was writing this then I suggested by Variable documentation - I thought that works in the same way ;) I mean that when you once declare variable then you can't overwrite these var by another var - maybe I'm wrong?

Reply:
I don't know, probable it works the same way.